### DISCUSSION TITLE
How to transfer CNAME domain to new project?

### DISCUSSION THREAD

**Discussion by gvholley** [community/community#22170] [Other Feature Feedback, Questions, & Ideas]
<p>I am using my main Github page (<a href="http://myname.github.io" rel="noopener nofollow ugc">myname.github.io</a>) as a portfolio, which redirects to a custom domain (<a href="http://myname.com" rel="noopener nofollow ugc">myname.com</a>). However, I made a completely new portfolio in a different repository which is currently live as a project page (<a href="http://myname.github.io/new_portfolio" rel="noopener nofollow ugc">myname.github.io/new_portfolio</a>). I want to use <a href="http://myname.com" rel="noopener nofollow ugc">myname.com</a> for my new portfolio project instead of my old one.</p>
<p>My CNAME is a Google domain, so it is linked to “<a href="http://myname.github.io" rel="noopener nofollow ugc">myname.github.io</a>.”. I am not sure what needs to be changed where. Do I need to change “<a href="http://myname.github.io" rel="noopener nofollow ugc">myname.github.io</a>.” to "<a href="http://myname.github.io/new_portfolio.%22" rel="noopener nofollow ugc">myname.github.io/new_portfolio."</a> in the Google Domain dashboard? Or do I need to somehow transfer <a href="http://myname.github.io" rel="noopener nofollow ugc">myname.github.io</a> to my new portfolio project? How can I do that through Github?</p>
<p>Basically I want my current Github user page to change to <a href="http://myname.github.io/old_portfolio" rel="noopener nofollow ugc">myname.github.io/old_portfolio</a> and <a href="http://mhname.github.io/new_portfolio" rel="noopener nofollow ugc">mhname.github.io/new_portfolio</a> to change to <a href="http://myname.github.io" rel="noopener nofollow ugc">myname.github.io</a></p>

**Comment by otter-computer** [comment, marked as answer]
<p>It should be <em>fairly</em> simple to do this but you’ll need to rename your repositories to do this.</p>
<p>First of all you’ll need to detach your custom domain from your current repository. You can do this by visiting the repository settings, editing the custom domain, and deleting it deom the input box.</p>
<p>Next you’ll need to <em>rename</em> this repository to something other than <code>gvholley.github.io</code>–perhaps rename it to <code>old-website</code> or something.</p>
<p>Finally, visit your new repository. You’ll want to rename this repository to <code>gvholley.github.io</code>. Then you’ll want to attach your domain name to this repo instead. You may need to refresh the settings page and enable GitHub Pages after renaming your repo, but you should be able to add it in the custom domain input.</p>
